sonar-project.properties

```diff
@@ -1,6 +1,6 @@
 sonar.projectKey=fh-mbaas
 sonar.projectName=fh-mbaas-nightly-master
-sonar.projectVersion=6.0.1
+sonar.projectVersion=6.0.2
```

Comment:
Can remove this file as not using Sonarqube any more

Reply:
Let's have a compromise - I'll remove it in my next PR. :)